handler/src/main/java/io/netty/handler/ssl/JdkSslContext.java

```diff
@@ -375,15 +375,21 @@ private SSLEngine configureAndWrapEngine(SSLEngine engine, ByteBufAllocator allo
                     throw new Error("Unknown auth " + clientAuth);
             }
         }
+        configureEndpointVerification(engine);
```

Comment:
A non-alpn case is already tested by `SocketStartTlsTest`. If I remove `endpointIdentificationAlgorithm(null)` from lines 93 and 103 it passes, but after my change it fails.